feature-libs/cart/main/components/cart/add-to-cart/add-to-cart.module.ts

```diff
@@ -37,19 +30,8 @@ import { AddedToCartDialogComponent } from './added-to-cart-dialog/added-to-cart
     KeyboardFocusModule,
     ModalModule,
   ],
```

Comment:
can we remove the unused imports?

Reply:
I removed some imports that didn't seem used and I refactored a bit to reflect thee fact that AddedToCart-dialogComponent is now alone in his module.